crit/mempages_test.go

```diff
@@ -248,6 +248,29 @@ func TestGetPsArgsAndEnvVars(t *testing.T) {
 	}
 }
 
+func TestGetShmemSize(t *testing.T) {
```

Comment:
@rst0git, for that purpose, would it be good to update the `loop` program by integrating the allocation of shared memory using `mmap()`? This could then be utilized within the `crit` integration test.

Reply:
It would be better to use a separate test program that takes as input *type* and *size* of shared memory. This would allow us to reuse the same test code with different size of memory and verify the output of `GetShmemSize()`.